package.json

```diff
@@ -34,15 +33,13 @@
   "devDependencies": {
     "adm-zip": "0.4.13",
     "azure-devops-node-api": "^9.0.1",
-    "azure-pipelines-task-lib": "2.9.3",
     "js-yaml": "^3.13.1",
     "markdown-toc": "^1.2.0",
     "minimatch": "3.0.2",
     "minimist": "^1.1.1",
     "mocha": "2.3.3",
     "mocha-junit-reporter": "1.17.0",
     "node-uuid": "1.4.6",
-    "@octokit/rest": "17.0.0",
```

Comment:
Octokit is the thing causing issues, its not compatible with older versions of typescript (for what seem to be typings issues).

An easy thing to do here to fix is to move it into the ci folder as its own thing (like [courtesy-push](https://github.com/microsoft/azure-pipelines-tasks/tree/master/ci/courtesy-push)) - arguably that's where it belongs anyways. Its odd in general that we're pinned to this old TS version, but there's other assumptions baked in here that are hard to quickly change. I'd like to not have that dependency eventually.

cc/ @stephenmichaelf we'll need a different solution here.